src/app/checkout/step-2/existingPaymentMethods/existingPaymentMethods.tpl.html

```diff
@@ -3,9 +3,9 @@
   <div class="panel-body">
     <div class="radio radio-method" ng-repeat="paymentMethod in $ctrl.paymentMethods" ng-init="expired = !$ctrl.validPaymentMethod(paymentMethod)">
       <label>
-        <input ng-if="!expired" type="radio" name="paymentMethod" ng-model="$ctrl.selectedPaymentMethod" ng-value="paymentMethod" required>
-        <button ng-if="expired" class="btn btn-xs btn-default" ng-click="$ctrl.openPaymentMethodFormModal(paymentMethod)">Update</button>
+        <input type="radio" name="paymentMethod" ng-model="$ctrl.selectedPaymentMethod" ng-value="paymentMethod" ng-disabled="expired" required>
         <payment-method-display payment-method="paymentMethod" expired="expired"></payment-method-display>
+        <button class="btn btn-xs btn-link" ng-click="$ctrl.openPaymentMethodFormModal(paymentMethod)" translate>update</button>
```

Comment:
I had this link as `update` initially and Jason requested it be changed to `edit`.

Maybe switch label depending on on if it's expired?